packages/types/src/cart/service.ts

```diff
@@ -136,4 +139,44 @@ export interface ICartModuleService extends IModuleService {
     selector: Partial<CartShippingMethodDTO>,
     sharedContext?: Context
   ): Promise<void>
+
+  listShippingMethodAdjustments(
+    filters: FilterableShippingMethodProps,
+    config?: FindConfig<ShippingMethodAdjustmentDTO>,
+    sharedContext?: Context
+  ): Promise<ShippingMethodAdjustmentDTO[]>
+
+  addShippingMethodAdjustments(
+    data: CreateShippingMethodAdjustmentDTO[]
+  ): Promise<ShippingMethodAdjustmentDTO[]>
+  addShippingMethodAdjustments(
+    data: CreateShippingMethodAdjustmentDTO
+  ): Promise<ShippingMethodAdjustmentDTO[]>
```

Comment:
```suggestion
  ): Promise<ShippingMethodAdjustmentDTO>
```

if the other comment above tracks